tools/codegen.mk

```diff
@@ -24,3 +26,19 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+
+define genProtoForTarget
+.PHONY: $(1)
+protoc-gen-$(1)-v1:
+	protoc -I . ./dapr/proto/$(1)/v1/*.proto --go_out=plugins=grpc:.
+	cp -R ./github.com/dapr/dapr/pkg/proto/$(1)/v1/*.go ./pkg/proto/$(1)/v1
+	rm -rf ./github.com
```

Comment:
we can use `mv` here for the replacement of cp and rm.  mv should work. 
```
mv ./github.com/dapr/dapr/pkg/proto/$(1)/v1/*.go ./pkg/proto/$(1)/v1
```

Reply:
mv will move the file only but keep the temp folder, while the current solution will end up removing the temp folder too in the end.